index.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+var debug = require('debug')('strong-pack');
+var fstream = require('fstream');
+var tar = require('tar');
+
+module.exports = exports = pack;
+
+var FILTER = filterize([
+  '.git',
+  'CVS',
+  '.svn',
+  '.hg',
+  '.lock-wscript',
+  /^\.wafpickle-[0-9]+$/,
+  /^\..*\.swp$/,
+  '.DS_Store',
+  /^\._/,
+]);
+
+function pack(folder) {
+  var tarPack = tar.Pack({ noProprietary: true });
+  var contents = fstream.Reader({
+    path: folder,
+    type: 'Directory',
+    isDirectory: true,
+    filter: FILTER,
+  });
+
+  return contents
+    .on('error', debug.bind(null, 'err reading folder'))
+    .pipe(tarPack)
+    .on('error', debug.bind(null, 'tar creation error'));
```

Comment:
This makes node think that errors have been "handled", but from the external user's point of view they are still "unhandled". Is that intentional? You could do `if (debug.enabled) contents.on('error', ...)`.